command.cc

```diff
@@ -5,7 +5,7 @@ int main(int argc, char** argv) {
         "localhost:50002", grpc::InsecureChannelCredentials()));
     string array[7];
     string curr;
-    cout<<"argc: "<<argc<<endl;
+    // cout<<"argc: "<<argc<<endl;
```

Comment:
I'd recommend just deleting this line.

Reply:
Yes that's fine, it just used to check if there are invalid commands

Reply:
done